src/envo/envo-edit.owl

```diff
@@ -12617,53 +12592,51 @@ AnnotationAssertion(owl:deprecated <http://purl.obolibrary.org/obo/ENVO_00000886
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000887> (large river biome)
 
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasOBONamespace> <http://purl.obolibrary.org/obo/ENVO_00000887> "ENVO"^^xsd:string)
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/ENVO_00000887> "ENVO:00000887"^^xsd:string)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000887> "A freshwater river biome which is determined by a large river."@en)
```

Comment:
i assume 'large river' is defined in ENVO?  what does 'determined by a river' mean?